x/feemarket/types/params.go

```diff
@@ -172,10 +172,13 @@ func validateBaseFeeChangeDenominator(i interface{}) error {
 }
 
 func validateElasticityMultiplier(i interface{}) error {
-	_, ok := i.(uint32)
+	value, ok := i.(uint32)
```

Comment:
I think the ParamSet is not used anymore, there's a more straightforward `Validate` method.